.buildkite/commands/build-for-windows.ps1

```diff
@@ -0,0 +1,49 @@
+# Accept a parameter to determine build type (required)
+param (
+    [Parameter(Mandatory=$true)]
+    [string]$BuildType
+)
+
+# Stop script execution when a non-terminating error occurs
+$ErrorActionPreference = "Stop"
+
+# Define constants for build types
+$BUILD_TYPE_DEV = "dev"
+$BUILD_TYPE_RELEASE = "release"
+$VALID_BUILD_TYPES = @($BUILD_TYPE_DEV, $BUILD_TYPE_RELEASE)
+
+# Validate build type
+if ($BuildType -notin $VALID_BUILD_TYPES) {
+    Write-Host "Error: BuildType must be one of: $($VALID_BUILD_TYPES -join ', ')" -ForegroundColor Red
+    exit 1
+}
```

Comment:
I like this check 😄 

Wonder if there's a difference between `exit 1` and `Exit 1`